behaviour/graql/reasoner/BUILD

```diff
@@ -18,6 +18,14 @@
 package(default_visibility = ["//visibility:public"])
 
 exports_files([
-    "resolution.feature",
-    "rule-validation.feature",
+    "attribute-attachment.feature",
+    "concept-inequality.feature",
+    "negation.feature",
+    "relation-inference.feature",
+    "roleplayer-attachment.feature",
+    "test-framework.feature",
```

Comment:
this should now become `test-resolution-framework` since the outer scope of this is now just `reasoner`

Reply:
Updated to `resolution-test-framework` - I was also debating internally whether to make it explicit.